x-pack/plugins/uptime/public/pages/page_header.tsx

```diff
@@ -58,21 +58,20 @@ export const PageHeader = React.memo(
       ) : null;
 
     const kibana = useKibana();
-    const history = useHistory();
 
     const extraLinkComponents = !extraLinks ? null : (
       <EuiFlexGroup alignItems="flexEnd" responsive={false}>
         <EuiFlexItem grow={false}>
           <ToggleAlertFlyoutButton />
         </EuiFlexItem>
         <EuiFlexItem grow={false}>
-          <EuiButtonEmpty
+          <EuiReactRouterButtonEmpty
```

Comment:
Likewise if we add an `aria-label` to the overview link add one here.

Reply:
same here, it's not necessary here and i don't think so it will add any value.